src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -349,6 +349,9 @@ public void halt(int status) {
      * @throws  IllegalArgumentException
      *          If {@code command} is empty
      *
+     * @implNote
+     * The new process creation is logged by {@link ProcessBuilder#start()}.
```

Comment:
I agree it is useful to have an implNote here but I don't think it words to just say that it is logged by PB.start. Instead I think it should say that the process created is logged, see PB.start for details.